include/jwt/impl/jwt.ipp

```diff
@@ -530,7 +530,7 @@ jwt_object::three_parts(const jwt::string_view enc_str)
 
   result[1] = jwt::string_view{&enc_str[fpos + 1], spos - fpos - 1};
```

Comment:
Similar check required here for fpos ?

Reply:
There is only one call to the function `three_parts` which is preceeded by a check that the string has at least two '.' characters: https://github.com/arun11299/cpp-jwt/blob/master/include/jwt/impl/jwt.ipp#L669-L677